extensions-core/azure-extensions/src/main/java/org/apache/druid/storage/azure/AzureDataSegmentPusher.java

```diff
@@ -107,7 +104,7 @@ public String getStorageDir(DataSegment dataSegment, boolean useUniquePath)
         useUniquePath ? DataSegmentPusher.generateUniquePath() : null
     );
 
-    log.info("DataSegment: [%s]", seg);
+    log.info("DataSegment Suffix: [%s]", seg);
```

Comment:
why do we want / need to change this to return suffix of storage dir instead of full path?

Reply:
this is needed because `pushToPath` accepts `storageDirSuffix` not the full path as it prefixes the path with configured prefix and this method is called in `push` method before calling `pushToPath`. Also I you see implementations of `getStorageDir` for other pushers they all return suffix so this change is inline with that.